rotkehlchen/fval.py

```diff
@@ -22,6 +22,9 @@ class FVal:
 
     def __init__(self, data: AcceptableFValInitInput = 0):
 
+        decimal_context = getcontext()
+        decimal_context.prec = 78
+        setcontext(decimal_context)
```

Comment:
Initializing a `FVal` is something that happens a lot and I don't know how expensive this operation is but it adds an overhead. I'm reading in the docs that this takes effect per thread so theorically you only need to execute it once when starting the app